readline/codes.go

```diff
@@ -51,8 +51,13 @@ var (
 	seqDelete2      = string([]byte{27, 91, 80})
 	seqCtrlDelete   = string([]byte{27, 91, 51, 59, 53, 126})
 	seqCtrlDelete2  = string([]byte{27, 91, 77})
+	seqAltDelete    = string([]byte{27, 91, 51, 59, 51, 126})
+	seqAltDelete2   = string([]byte{27, 91, 80}) // idk why this doesnt work in st
```

Comment:
for some reason this doesnt work in st even though `showkey -a` says thats what i am pressing and C-v inside of bash/zsh say give out ^[[P (which is 27 for escape, 91 for [ and 80 for P in ascii